scripts/generate_json_schemas.sh

```diff
@@ -27,6 +29,7 @@ LINKML_ITEMS=(
   "address.yaml,PostalAddressClass,PostalAddress.json"
   "document.yaml,ResidencePermitDetailsClass,ResidencePermit.json"
   "riskAssessmentCredential.yaml,RiskAssessmentCredentialClass,RiskAssessmentCredential.json"
+  "credentials.yaml,RiskAssessmentCredentialJWTClass,RiskAssessmentCredentialJWT.json"
   "securityCheckCredential.yaml,SecurityCheckCredentialClass,SecurityCheckCredential.json"
```

Comment:
IMO we should remove all the inner JSON schemas and only generate schemas for the top-level credentials (i.e. the JWTs), but I don't want to remove any schemas that people may be relying on.